boards/arm/tms570/launchxl-tms57004/scripts/Make.defs

```diff
@@ -25,7 +25,6 @@ include $(TOPDIR)/arch/arm/src/armv7-r/Toolchain.defs
 LDSCRIPT = flash-sram.ld
 ARCHSCRIPT += $(BOARD_DIR)$(DELIM)scripts$(DELIM)$(LDSCRIPT)
 
-ARCHCPUFLAGS = -mcpu=cortex-r4 -mbig-endian
```

Comment:
I think TMS570 is the only Arch on NuttX that supports Big Endian, it should be a nice platform to verify if there are issues in the system and/or drivers. Is the big-endian passed in other place now or it will assume that ARCH will be little-endian?

Reply:
The problem is that we can't find the prebuilt toolchain which ship with the big endian library. @acassis can you find one? @anchao can enable tms570 in Ci, if you can fine one.

Reply:
The config still exist: https://bitbucket.org/nuttx/buildroot/src/master/configs/ so we can build it. I will try to find the board to buy and test it. Seems it still available on ti.com website